packages/ffe-feedback-react/package.json

```diff
@@ -0,0 +1,53 @@
+{
+    "name": "@sb1/ffe-feedback-react",
+    "version": "1.0.0",
+    "description": "React component for ffe-feedback",
+    "license": "MIT",
+    "author": "SpareBank 1",
+    "files": [
+        "lib",
+        "es",
+        "types"
+    ],
+    "main": "lib/index.js",
+    "module": "es/index.js",
+    "types": "types/index.d.ts",
+    "sideEffects": false,
+    "repository": {
+        "type": "git",
+        "url": "ssh://git@github.com:SpareBank1/designsystem.git"
+    },
+    "scripts": {
+        "build": "ffe-buildtool babel",
+        "watch": "ffe-buildtool babel-watch",
+        "lint": "eslint src",
+        "test": "ffe-buildtool jest",
+        "test:watch": "ffe-buildtool jest --watch"
+    },
+    "dependencies": {
+        "@sb1/ffe-buttons-react": "^20.0.4",
+        "@sb1/ffe-collapse-react": "^2.0.6",
+        "@sb1/ffe-core-react": "^7.1.4",
+        "@sb1/ffe-feedback": "^1.0.0",
+        "@sb1/ffe-form": "^25.0.2",
```

Comment:
Du trenger ikke ha dependency til "ffe-form", fordi den er indrekte med gjennom at den er en dependency i ffe-form-react

Reply:
Okei, det visste jeg ikke! Da skal jeg forenkle i `package.json` :relaxed: 

Gjelder det for øvrig i appene der disse bibliotekene brukes også, at dersom man har avhengighet til react-pakken, så trenger man ikke eksplisitt avhengighet til less-pakken? (Men kan fremdeles importere less-filen ettersom less-pakken finnes i node-modules?)